pkg/autoscaler/autoscaler.go

```diff
@@ -125,10 +125,10 @@ type Autoscaler struct {
 }
 
 // New creates a new instance of autoscaler
-func New(dynamicConfig *DynamicConfig, model v1alpha1.RevisionRequestConcurrencyModelType, reporter StatsReporter) *Autoscaler {
+func New(dynamicConfig *DynamicConfig, containerConcurrency v1alpha1.RevisionContainerConcurrencyType, reporter StatsReporter) *Autoscaler {
```

Comment:
These lines are getting long enough that they get cut off during GitHub code review (120 chars).

Reply:
Formatted onto multiple lines.